modules/ocf_stats/manifests/munin.pp

```diff
@@ -38,16 +43,13 @@
   include apache::mod::fcgid
   include apache::mod::rewrite
 
-  $cname = $::host_env ? {
-    'dev'  => 'dev-munin',
-    'prod' => 'munin',
-  }
-
   apache::vhost {
     'munin':
       servername    => "${cname}.ocf.berkeley.edu",
       port          => 443,
       docroot       => '/var/cache/munin/www/static/',
+      docroot_owner => 'munin',
+      docroot_group => 'munin',
```

Comment:
Is there a reason these aren't currently set?

Reply:
I don't believe so? Not sure how it's currently set up though.

Reply:
@chriskuehl ^